packages/components/organisms/f-header/src/tenants/it-IT.js

```diff
@@ -18,7 +18,7 @@ export default {
             gtm: 'click_account_your_orders'
         },
         accountCredit: {
-            text: 'Credito dell',
+            text: "Credito dell'account",
```

Comment:
You should be able to escape the `'` with a backslash so you can keep the single quotes.

Reply:
Do you have any policies or guidelines on this? In f-footer we use the format I used here. And as for me, it's more readable anyway.